src/SWP/Bundle/AnalyticsBundle/Model/RequestMetric.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\AnalyticsBundle\Model;
+
+/**
+ * RequestMetric -.
+ */
+class RequestMetric
+{
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var @string
+     */
+    protected $uri;
+
+    /**
+     * @var @string
+     */
+    protected $route;
+    /**
+     * @var @int
+     */
+    protected $duration;
+
+    /**
+     * @var \DateTime
+     */
+    protected $created;
+
+    /**
+     * Get id.
+     *
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set uri.
+     *
+     * @param string $uri
+     *
+     * @return RequestMetric
+     */
+    public function setUri($uri)
+    {
+        $this->uri = $uri;
+
+        return $this;
+    }
+
+    /**
+     * Get uri.
+     *
+     * @return string
+     */
+    public function getUri()
+    {
+        return $this->uri;
+    }
+
+    /**
+     * Set route.
+     *
+     * @param string $route
+     *
+     * @return RequestMetric
+     */
+    public function setRoute($route)
+    {
+        $this->route = $route;
+
+        return $this;
+    }
+
+    /**
+     * Get route.
+     *
+     * @return string
+     */
+    public function getRoute()
+    {
+        return $this->route;
+    }
+
+    /**
+     * Get duration.
+     *
+     * @return string
+     */
+    public function getDuration()
+    {
+        return $this->duration;
+    }
+
+    /**
+     * Set duration.
+     *
+     * @param int $duration
+     *
+     * @return RequestMetric
+     */
+    public function setDuration($duration)
+    {
+        $this->duration = $duration;
+
+        return $this;
```

Comment:
do you really need a fluent interface here? If you don't use it anywhere in the bundle then just `$this->duration = $duration;` is enough, pls check also in other places